### PR TITLE
added integration instructions to getting started

### LINKED ISSUE #92: Docs: Add instructions on how to use PyAirbyte with LangChain
We don't have an entry in our own docs yet for the LangChain integration.

There are really two ways to integrate...

## LangChain native integration (`airbyte-langchain`)

Erick shows this really well:

- LinkedIn: https://www.linkedin.com/posts/langchain_universal-document-loader-with-langchain-airbyte-activity-7169410958346444801-6hYa/
- Docs: https://python.langchain.com/docs/integrations/document_loaders/airbyte
- YouTube: https://www.youtube.com/watch?v=zQU_1sCLSMU

## PyAirbyte-centric integration

You can also directly use PyAirbyte to create documents and send them to LangChain without importing `airbyte-langchain`.

Example here:

https://colab.research.google.com/drive/1IiP224qBttdSVYA31mutgokDsMk-1DAq#scrollTo=I7COppv6yGpv

### PR DESCRIPTION
**Pull request by bindipankhudi**
Resolves https://github.com/airbytehq/PyAirbyte/issues/92

This PR adds Langchain integration section to to the "Using PyAirbyte" page. The content is taken from related ticket. Once this is in, I will add a link to this from readme in our repo.

### CODE REVIEW

File: docs/using-airbyte/pyairbyte/getting-started.mdx [hunk 1]
```diff
@@ -70,3 +70,11 @@ PyAirbyte is a python library that can be run in any context that supports Pytho
...
 
 <AirbyteLibConnectors />
+
+## LangChain integration
+
+For those interested in using PyAirbyte to drive your LLM use cases, we provide two ways to integrate with LangChain:
+
+* **LangChain native integration**: This approach requires you to utilize the langchain-airbyte integration package. Refer to [LangChain Docs](https://python.langchain.com/docs/integrations/document_loaders/airbyte) or watch this [YouTube video](https://www.youtube.com/watch?v=zQU_1sCLSMU) to get started.
+
+* **PyAirbyte-centric integration**: You can also directly use PyAirbyte to create documents. With this approach, you do not need to import langchain-airbyte. Refer to [PyAirbyte Document Creation Demo](https://colab.research.google.com/drive/1IiP224qBttdSVYA31mutgokDsMk-1DAq#scrollTo=I7COppv6yGpv) to get started.
```
Comment (aaronsteers) [inline docs/using-airbyte/pyairbyte/getting-started.mdx:80]: This notebook link is not necessarily stable. Wdyt of us moving a version of this notebook into our Quickstarts? Or else just pasting the code inline here or on another page.

File: docs/using-airbyte/pyairbyte/getting-started.mdx [hunk 2]
```diff
@@ -70,3 +70,11 @@ PyAirbyte is a python library that can be run in any context that supports Pytho
 The following connectors are available:
 
 <AirbyteLibConnectors />
+
+## LangChain integration
+
+For those interested in using PyAirbyte to drive your LLM use cases, we provide two ways to integrate with LangChain:
+
+* **LangChain native integration**: This approach requires you to utilize the langchain-airbyte integration package. Refer to [LangChain Docs](https://python.langchain.com/docs/integrations/document_loaders/airbyte) or watch this [YouTube video](https://www.youtube.com/watch?v=zQU_1sCLSMU) to get started.
```
Comment (aaronsteers) [inline docs/using-airbyte/pyairbyte/getting-started.mdx:78]: Minor nit. Surrounding the library name with backticks will make it more clear this is an exact package name. 

```suggestion
* **LangChain native integration**: This approach requires you to utilize the `langchain-airbyte` integration package. Refer to [LangChain Docs](https://python.langchain.com/docs/integrations/document_loaders/airbyte) or watch this [YouTube video](https://www.youtube.com/watch?v=zQU_1sCLSMU) to get started.
```

**Review by aaronsteers — APPROVED** [review]
Approving, with a couple small comments.

It's a bit more work to do so, but I'd feel more comfortable if we had the Colab notebook committed to the quickstarts repo (as we do with the others) - in order to ensure it is a stable reference. Also, that notebook has a bunch of comments that were useful during planning, but are a bit cumbersome now for new users who just want to learn the integration.

File: docs/using-airbyte/pyairbyte/getting-started.mdx (hunk 2 above)
Comment (bindipankhudi) [reply docs/using-airbyte/pyairbyte/getting-started.mdx:78]: will do that.

File: docs/using-airbyte/pyairbyte/getting-started.mdx (hunk 1 above)
Comment (bindipankhudi) [reply docs/using-airbyte/pyairbyte/getting-started.mdx:80]: yea i was wondering the same. let me clean and up and commit to quickstarts first, that way can add the link to top of the page too.

**Comment by natikgadzhi** [comment]
Oh, so reference `pyairbyte` docs are in their own repo, but usage docs are in the main docs site, right?

**Comment by bindipankhudi** [comment]
Yea, the docs are still in the main repo and linked from the PyAirbyte
repo.

File: docs/using-airbyte/pyairbyte/getting-started.mdx (hunk 1 above)
Comment (bindipankhudi) [reply docs/using-airbyte/pyairbyte/getting-started.mdx]: Done.

File: docs/using-airbyte/pyairbyte/getting-started.mdx (hunk 2 above)
Comment (bindipankhudi) [reply docs/using-airbyte/pyairbyte/getting-started.mdx]: done.

**Pull request merged**